server.js

```diff
@@ -1,14 +1,46 @@
+/*
+make this a simple http server that writes to the response stream object
 
-// make this a simple http server that writes to the response stream object
+the page should display the following message:
 
-// the page should display the following message:
+Today is <day name>, <month name> <date>. Here are the available food trucks:
 
-// Today is <day name>, <month name> <date>. Here are the available food trucks:
+list the food trucks returned by filterByDay, passing in the current day name.
+e.g. filterByDay(day); where "day" is determined using the enhanced-date module
+The list of trucks returned will be an array of food truck objects. Iterate
+through the list, building up a string of food truck names. Once you're done
+iterating through that list, display the string you built up.
 
-// list the food trucks returned by filterByDay, passing in the current day name.
-// e.g. filterByDay(day); where "day" is determined using the enhanced-date module
-// The list of trucks returned will be an array of food truck objects. Iterate 
-// through the list, building up a string of food truck names. Once you're done 
-// iterating through that list, display the string you built up.
+Remember that the response is a stream object that must be closed.*/
 
-// Remember that the response is a stream object that must be closed.
\ No newline at end of file
+var http = require('http');
+
+http.createServer(function (request, response) {
+  //bind node modules to the following variables:
+  var dateObj = require('./enhanced-date');
```

Comment:
Best practice is to require your modules at the top of the file. Since this is a function that is executed each time the server handles a request, it will evaluate the require calls needlessly each time.